datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -113,6 +115,14 @@ public FlowIdentifier triggerSdxResize(Long sdxClusterId, SdxCluster newSdxClust
                 environmentClientService.getBackupLocation(newSdxCluster.getEnvCrn()), performBackup));
     }
 
+    public FlowIdentifier triggerSdxResizeRecovery(SdxCluster oldSdxCluster, SdxCluster newSdxCluster) {
```

Comment:
I don't see this method been used anywhere. Is this feature not finished?

Reply:
It's not that this is not finished, but that the goal of this ticket was to create the recovery flow chain as well as a trigger hook for it. This will be expanded upon in the future.

Reply:
triggerSdxResizeRecovery should be triggered from ResizeRecoveryService. This PR should the invocation as well.

Reply:
@kkalvagadda1 Dan's current change kind of mixes with this, should we not wait for either of these changes to be merged before making that modification? I don't want to run into merge conflicts.

Reply:
Hi, how could you guarantee that this pr will work if just call the trigger method? I think we must wait until you can test and call the trigger. It will easier to fix before the merge than after

Reply:
I tied in the recovery service.